pkg/kubelet/images/image_manager.go

```diff
@@ -85,7 +87,15 @@ func (m *imageManager) EnsureImageExists(pod *api.Pod, container *api.Container,
 		glog.Errorf("Couldn't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
 	}
 
-	spec := kubecontainer.ImageSpec{Image: container.Image}
+	// If the image contains no tag or digest, a default tag should be applied.
+	image, err := applyDefaultImageTag(container.Image)
```

Comment:
I think rkt assumes "latest" by default. https://github.com/kubernetes/kubernetes/blob/b1ee54346347f3675f285d24e302d36980983804/pkg/kubelet/rkt/image.go#L134

Instead of letting each runtime sets a different default, it makes more sense for kubelet to set the default to ensure consistent user experience.